tutorials/tutorial_07_mqtt_publisher/README.md

```diff
@@ -1,4 +1,4 @@
-# Tutorial 8 - MQTT Publisher
+# Tutorial 7 - MQTT Publisher
 
 This tutorial shows you how to communicate between apps on the local network.
```

Comment:
This is probably wrong, will need to be addressed when we do a webrtc sample probably

Reply:
Updated to

> This tutorial shows you how to communicate between apps **through ZED Hub**.